homeassistant/helpers/entity_registry.py

```diff
@@ -703,6 +706,19 @@ async def async_load(self) -> None:
 
         from .entity import EntityCategory  # pylint: disable=import-outside-toplevel
 
+        def parse_enum(
```

Comment:
We should not define that as inline function, let's place that as standalone function into that file